docs/streams/architecture.html

```diff
@@ -80,6 +80,10 @@ <h3><a id="streams_architecture_tasks" href="#streams_architecture_tasks">Stream
         tasks will be automatically restarted on other instances and continue to consume from the same stream partitions.
     </p>
 
+    <p><b>NOTE:</b> Topic partitions are balanced but may be unevenly distributed across consumers. Kafka Streams uses the
```

Comment:
Not sure what `are balanced but may be unevenly distributed across consumers` means?

Reply:
I'm not sure either! :) It's what we ended up with in https://github.com/confluentinc/docs/pull/3604.

Reply:
@JimGalasyn Can we update this to (also in CP docs):
```
Topic partitions are assigned to tasks, and tasks are assigned to all threads over all instances in an best-effort manner try to trade-off load-balancing and stickiness of stateful tasks. For this assignment, Kafka Streams uses ...
```

Reply:
Much better, thanks!